### ISSUE TITLE
Inclusion criteria

### ISSUE THREAD

**Issue by RichardRed0x** [RichardRed0x/decred-media-tracker#11]
There are too many junk articles out there to include all of them in this sheet. We need some criteria for inclusion. I see this as part of the "source/publication assessment" (issue #6 ) question, effectively below a certain "source quality" level the article (or other media) is not even included in the sheet.

Here's an example of something that would fall below the threshold: https://icobrothers.media/2018/10/24/decred-price-changed-by-5-95-percent/

icobrothers.media publishes an "article" like this every day

**Comment by xaur** [comment]
Yeah that is a good example of an entry that would not benefit at all from inclusion, except just a few times to demonstrate the pattern.

Generally generated content like this is not interesting for study, but I see value in tracking junk (co-)written by humans e.g. http://www.hotstocked.com/article/94031/is-decred-a-bitcoin-alternative-or-a-scam.html -- they even deleted my comment so I'm sure humans are behind this.
